### PR TITLE
Update samples

### PR DESCRIPTION
**Pull request by sturkmen72**
### This pullrequest changes

updates samples

```
force_builders_only=linux,windows,docs
```

### CODE REVIEW

File: samples/cpp/peopledetect.cpp [hunk 1]
```diff
@@ -5,7 +5,6 @@
 #include <opencv2/objdetect.hpp>
 #include <opencv2/highgui.hpp>
 #include <opencv2/imgproc.hpp>
-#include <opencv2/videoio.hpp>
```
Comment (alalek) [inline samples/cpp/peopledetect.cpp]: Probably we should keep separate include of "videoio" module - `VideoCapture` is used below.
(due compatibility "videoio" headers are included by "highgui" header, but this doesn't look consistent - lets show to Users which modules are used)

Comment (sturkmen72) [reply samples/cpp/peopledetect.cpp]: @alalek OK. let me add `#include <opencv2/videoio.hpp>` in all samples uses `VideoCapture`
please don't merge until i inform you.

**Comment by alalek** [comment]
@sturkmen72 Could you please take a look on build failures?

**Comment by sturkmen72** [comment]
@alalek i think the PR is ready.

**Pull request merged**